conf/test_hybrid_host_rm.config

```diff
@@ -1,25 +1,29 @@
 /*
- * -------------------------------------------------
- *  Nextflow config file for running tests
- * -------------------------------------------------
- * Defines bundled input files and everything required
- * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/mag -profile test_hybrid_host_rm
- */
+========================================================================================
+    Nextflow config file for running minimal tests
+========================================================================================
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/mag -profile test_hybrid_host_rm,<docker/singularity>
+
+----------------------------------------------------------------------------------------
+*/
 
 params {
-  config_profile_name = 'Test profile'
-  config_profile_description = 'Minimal test dataset to check pipeline function'
-  // Limit resources so that this can run on GitHub Actions
-  max_cpus = 2
-  max_memory = 6.GB
-  max_time = 48.h
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = 6.GB
+    max_time   = 48.h
 
-  // Input data
-  host_fasta = "https://github.com/nf-core/test-datasets/raw/mag/host_reference/genome.hg38.chr21_10000bp_region.fa"
-  input = 'https://github.com/nf-core/test-datasets/raw/mag/samplesheets/samplesheet.hybrid_host_rm.csv'
-  min_length_unbinned_contigs = 1
-  max_unbinned_contigs = 2
-  busco_reference = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
-  gtdb = false
+    // Input data
+    host_fasta                  = "https://raw.githubusercontent.com/nf-core/test-datasets/mag/host_reference/genome.hg38.chr21_10000bp_region.fa"
+    input                       = 'https://raw.githubusercontent.com/nf-core/test-datasets/mag/samplesheets/samplesheet.hybrid_host_rm.csv'
+    min_length_unbinned_contigs = 1
+    max_unbinned_contigs        = 2
+    busco_reference             = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
```

Comment:
I think you wrote that earlier, but why again `https://github.com/nf-core/test-datasets/raw/` to `https://raw.githubusercontent.com/nf-core/test-datasets` ?

Reply:
The template does it like this and I remembered someone saying this should be more robust or so